docs/resources/dns_auth_zone.md

```diff
@@ -19,7 +19,7 @@ resource "bloxone_keys_tsig" "example_tsig" {
 
 resource "bloxone_dns_acl" "example_acl" {
   name = "example_acl"
-  elements = [
+  list = [
```

Comment:
Thanks for this fix. Could have been a separate PR :)

Reply:
Noted. Will keep in mind from next time !